src/celestia/view.cpp

```diff
@@ -124,15 +113,10 @@ View::walkTreeResizeDelta(View* v, float delta, bool check)
             return false;
         if (check)
             return true;
-        v->height = (float) newSize;
+        v->height = newSize;
+        v->y = p->y + (v->y - p->y) * ratio;
         if (sign == 1)
```

Comment:
based on the old code here it should be "sign != 1"

Reply:
oh, when i stop doing dump mistakes?